tests/main/install-errors/task.yaml

```diff
@@ -61,3 +61,21 @@ execute: |
         exit 1
     fi
     MATCH 'cannot install "ubuntu-core", please use "core" instead' < stderr.out
+
+    echo "============================================"
+
+    echo "Install a snap that is only available in the edge channel"
+    if snap install test-snapd-only-in-edge 2>stderr.out; then
+        echo "Installing test-snapd-only-in-edge should fail but did not"
+        exit 1
+    fi
+    MATCH 'error: snap not found in the given context.' < stderr.out
```

Comment:
The error message is still confusing. Shouldn't we suggest using `snap info <foo>` to see the details of the snap?